docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+---
+id: data-types
+title: Data Types Overview
+sidebar_label: Data Types and File Formats
+---
+
+ICGC ARGO provides access to harmonized data from new programs
+
+## Reads
+
+### Submitted Reads
+
+Raw reads submitted by ARGO member programs that are used as inputs to the ARGO analytic pipeline(s). Data is accepted in FASTQ or BAM format and must be compressed.
```

Comment:
I think this needs rewording..

`Raw reads submitted by ARGO member programs are used as inputs within the ARGO analytic pipeline.`